apps/src/p5lab/spritelab/blocks.js

```diff
@@ -336,7 +336,12 @@ export default {
       },
       removeVar: Blockly.Blocks.variables_get.removeVar
     };
-    generator.sprite_variables_get = generator.variables_get;
+    generator.sprite_variables_get = function() {
+      return [
+        `{name: '${this.getTitleValue('VAR')}'}`,
+        Blockly.JavaScript.ORDER_ATOMIC
```

Comment:
Is this used to set the order of operations? I'm a bit nervous about setting it to the lowest value - it prevents us from setting anything lower.

Reply:
Yes, this sets the order of operations. I don't quite understand your concern, though. In what circumstance would we want it lower? I believe ATOMIC is the correct order here, because this is a standalone object value (as opposed to being some sort of operation). Also this change doesn't actually change the order- see https://github.com/code-dot-org/blockly/blob/master/generators/javascript/variables.js#L34

Reply:
Makes sense. 👍